### ISSUE TITLE
Analyze Sentences

### ISSUE THREAD

**Issue by Thomascountz** [Thomascountz/relay#24]
As a user,
When I click "Analyze",
I can see the results of the tone analysis for individual sentences in my document

**Comment by Thomascountz** [comment]
There’s a little glitch in the original plan for highlighting sentences to report the NPL analysis…

*tl;dr* Text highlighting is really hard within a textarea. Either a solution is fragile or too config-heavy. Can I come up with another UI/UX so as to keep the ball rolling?

___

`textarea` content can not be styled and I haven’t been able to find any good solutions.

One solution is to create a mirror `div` behind the `textarea` and highlight text there through inline styling. If you make the background of the `textarea` transparent, boom! You’re highlighted. [1]. The problem with this approach is that it’s proving to be fragile and doesn’t fit well within the React paradigm.

The second approach would be to use an external text editor framework such as draft.js [2]. This is a highly appropriate option, considering that it, like React, was created by Facebook, however, it’s a heavy-weight solution and not necessarily optimized for our use case. In future iterations, though, I would take another look at this, it’s really awesome!

The final option is to not highlight text at all, but rather display the results in a different way. The original product vision included the use of text highlighting, but I am reconsidering whether or not this feature will provide us with more business value than another solution could, during this MVP cycle.

I would recommend continuing driving out more of the core functionality by employing a simpler user interface, and relying heavily on our BOUNTIFUL user-feedback channels during upcoming iterations in order to get a feel for if we should continue with this feature.

[1] https://codersblock.com/blog/highlight-text-inside-a-textarea/
[2] https://draftjs.org/
